src/subdomains/core/payment-link/services/payment-link-payment.service.ts

```diff
@@ -55,6 +55,12 @@ export class PaymentLinkPaymentService {
 
   // --- JOBS --- //
   async processExpiredPayments(): Promise<void> {
+    await this.doProcessExpiredPayments();
+    await this.paymentActivationService.processExpiredActivations();
+    await this.paymentQuoteService.processExpiredQuotes();
+  }

```

Comment:
The code below will already cancel all quotes/activations for the expired payment. No need to process them separately. Keep this only for the cron job.
```ts
      await this.doSave(payment.expire(), true);
      await this.cancelQuotesForPayment(payment);
```
-> Keep the cron job as it is (with prolonged execution period)
-> Just call the code above when a payment expires (maybe move to a method)